wiki.moinmoin

```diff
@@ -380,15 +379,51 @@ RULE:     - the owning team will provide timely, high quality updates for the
+RULE:     - Such updates in the project might be trivial, but imply that a
+RULE:       dependency for e.g. a CVE fix will be moved to a new major version.
+RULE:       Being vendored that does gladly at least not imply incompatibility
+RULE:       issues with other packages or the SRU policy. But it might happen
+RULE:       that this triggers either:
+RULE:       a) The need to adapt the current version of the main package and/or
+RULE:          other vendored dependencies to work with the new dependency
+RULE:       b) The need to backport the fix in the dependency as the main
+RULE:          package will functionally only work well with the older version
+RULE:       c) The need to backport the fix in the dependency, as it would imply
+RULE:          requiring a newer toolchain to be buildable that isn't available
+RULE:          in the target release.
+RULE: - The rust ecosystem currently isn't yet considered stable enough for
+RULE:   classic lib dependencies and transitions in main; therefore the
+RULE:   expectation for those packages is to vendor (and own/test) all
+RULE:   dependencies (except those provided by the rust runtime itself).
+RULE:   This implies that all the rules for vendored builds always
+RULE:   apply to them. In addition
+RULE:   - Rust builds using librust-*-dev packges will populate the attribute
+RULE:     `X-Cargo-Built-Using`, right now this is expected not to be present.
+RULE:   - It is expected rust builds will use dh-cargo so that a later switch
+RULE:     to proper package dependencies isn't too hard (e.g. it is likely
+RULE:     that over time some more common/stable libs shall come from proper
+RULE:     archive packages).
+RULE: - All vendored dependencies (no matter what language) shall have a
+RULE:   way to be refreshed (recommended is a documentation README.source and
+RULE:   scripts in debian/
 TODO-A: - This does not use static builds
 TODO-B: - The team <TBD> is aware of the implications by a static build and
 TODO-B:   commits to test no-change-rebuilds and to fix any issues found for the
 TODO-B:   lifetime of the release (including ESM)
+
 TODO-A: - This does not use vendored code
 TODO-B: - The team <TBD> is aware of the implications of vendored code and (as
-TODO-B:   alerted by the security team) commits to provide updates to the security
-TODO-B:   team for any affected vendored code for the lifetime of the release
-TODO-B:   (including ESM).
+TODO-B:   alerted by the security team) commits to provide updates and backports
+TODO-B:   to the security team for any affected vendored code for the lifetime
+TODO-B:   of the release (including ESM).
+
+TODO-A: - This package is not rust based
+TODO-B: - This package is rust based and vendors all non language-runtime
+TODO-B:   dependencies
+
+TODO-A: - This package is not rust based
+TODO-B: - This package is rust based, refreshing vendored code is outlined
+TODO-B:   in <TBD>
```

Comment:
This is not rust specific, as mirrored in the review template.

Reply:
Indeed, merged with the above "This does not use vendored code"